tests/test_parser.py

```diff
@@ -6,7 +6,16 @@ def test_basic_run(sphinx_run, file_regression):
     sphinx_run.build()
     # print(sphinx_run.status())
     assert sphinx_run.warnings() == ""
-    assert sphinx_run.app.env.metadata == {"basic_run": {"test_name": "notebook1"}}
+    assert set(sphinx_run.app.env.metadata["basic_run"].keys()) == {
+        "test_name",
+        "kernelspec",
+        "language_info",
+    }
+    assert sphinx_run.app.env.metadata["basic_run"]["test_name"] == "notebook1"
+    assert (
+        sphinx_run.app.env.metadata["basic_run"]["kernelspec"]
+        == '{"display_name": "Python 3", "language": "python", "name": "python3"}'
```

Comment:
note - black did this, and it seemed reasonable + removed the line too long issues, so I went with it